comp/metadata/inventoryhaagent/def/component.go

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+// Package inventoryhaagent implements a component to generate the 'ha_agent_metadata' metadata payload for inventory.
+package inventoryhaagent
+
+// team: ndm-core
+
+// Component is the component type.
+type Component interface {
+	// GetAsJSON returns the payload as a JSON string. Useful to be displayed in the CLI or added to a flare.
+	GetAsJSON() ([]byte, error)
```

Comment:
Is this used anywhere ? Those methods are present in other payloads because they date back before we had the status components. We still need to clean the other payload now that each component embed its status.

Reply:
Hum, I see, just removed both Get and GetAsJSON and everything seems to work fine.